index.js

```diff
@@ -4,23 +4,85 @@ var fs = require('fs');
 var backlightPath = '/sys/class/backlight/rpi_backlight';
 
 function writeValue(fileName, value) {
-    return new Promise(function (resolve, reject) {
-        var fullPath = path.join(backlightPath, fileName);
-        fs.writeFile(fullPath, value, function(err) {
-             if (err !== null) reject(err);
-             else resolve();
-        });
+  return new Promise((resolve, reject) => {
+    var fullPath = path.join(backlightPath, fileName);
+    fs.writeFile(fullPath, value, (err) => {
+      if (err !== null) reject(err);
+      else resolve();
     });
+  });
+}
+
+function readValue(fileName, value) {
+  return new Promise((resolve, reject) => {
+    var fullPath = path.join(backlightPath, fileName);
+    fs.readFile(fullPath, 'utf8', (err, data) => {
+      if (err !== null){
+        reject(err); 
+      } else {
+        resolve(data);
+      }
+    });
+  });
 }
 
 if (!fs.existsSync(backlightPath)) {
-    throw new Error('Backlight control not supported (' + backlightPath + ' does not exist)');
+  throw new Error('Backlight control not supported (' + backlightPath + ' does not exist)');
 }
 
-exports.powerOn = function () {
-    return writeValue('bl_power', '0');
+/** Power managment */
+exports.powerOn = () => {
+  return writeValue('bl_power', '0');
+};
+
+exports.powerOff = () => {
+  return writeValue('bl_power', '1');
+};
+
+exports.getActualPowerValue = () => {
+  return readValue('bl_power');
+};
+
+/** Brightness managment */
+exports.setMaxBrightness = () => {
+  return writeValue('brightness', '255');
```

Comment:
255 isn't actually guaranteed to be the max brightness, the right value for the currently used screen is actually available in `/sys/class/backlight/rpi_backlight/max_brightness`. This function should read that, and then set the value it finds there.